src/main/java/net/neoforged/neoforge/registries/datamaps/builtin/NeoForgeDataMaps.java

```diff
@@ -90,6 +103,7 @@ private static ResourceLocation id(final String name) {
     private static void register(final RegisterDataMapTypesEvent event) {
         event.register(COMPOSTABLES);
         event.register(FURNACE_FUELS);
+        event.register(RAID_HERO_GIFTS);
```

Comment:
The field order in this class and the registration order are intentionally in alphabetical order, please change your additions to match that.

Reply:
fixed!

Reply:
The `RAID_HERO_GIFTS` field needs to be moved above the `VIBRATION_FREQUENCIES` field as well, not just the registration order.

Reply:
fixed!